viz/com.raytheon.uf.viz.core.maps/localization/bundles/maps/Aviation/ARTCC.xml

```diff
@@ -33,13 +33,13 @@
                             <capability xsi:type="outlineCapability"
                                 lineStyle="SOLID" />
                             <capability xsi:type="colorableCapability"
-                                colorAsString="#9b9b9b" />
+                                colorAsString="#FFC0CB" />
                         </capabilities>
                     </loadProperties>
                     <properties isMapLayer="true" />
                     <resourceData xsi:type="dbMapResourceData">
                         <table>mapdata.artcc</table>
-                        <mapName>ARTCC Areas</mapName>
+                        <mapName>Air Route Traffic Control Areas</mapName>
```

Comment:
Technically it's the Center... "Air Route Traffic Control Centers Areas"... although that sounds kinda messy.  Maybe just "Air Route Traffic Control Centers (ARTCCs)" as the title?
Because in aviation ARTCC is a known abbreviation... and the fact they're "Centers" is actually important...

Reply:
Yeah, I was honestly just changing it to what we had it in V18, but I can change it to "Air Route Traffic Control Centers" - it's already the longest menu item but I had to look up what ARTCC was cuz I didn't know

Reply:
Yeah I figured the length was probably the main motivator for changing it...
Is it in the Airport submenu though? So, you don't see it by default in the regular menu?  That might make it not as annoying that it's so long?